Symbolic Link Command Fix

jnich3ls0n opened the pull request:
(#640) All relative symbolic link commands are currently broken and receive the response `bash: command not found: x` in the honeypot shell.  The path is not being traversed correctly and fs.HoneyPotFilesystem.getfile() appears to be the source of problem.  Removing the cwd join command resolves the issue but I am not aware of other potential side effects.

The problem was first noted with `sh`.  I made `echo` a symbolic link and it failed in the same way.  Applying this fix corrected both on my system.

jnich3ls0n committed:
Symbolic Link Command Fix

All relative symbolic link commands are currently broken and receive the response "bash: command not found: x" in the honeypot shell.  The path is not being traversed correctly and fs.HoneyPotFilesystem.getfile() appears to be the source of problem.  Removing the cwd join command resolves the problem but I am not aware of other potential side effects.

micheloosterhof commented:
I think you're on the right track, but not sure about side effects yet. So let's test this out and see if it affects anything else.

jnich3ls0n commented:
I ran unit tests with the update and here are my results:

dist fs.py + dist fs.pickle = no errors

dist fs.py + createfs generated fs.pickle = errors for commands poweroff, reset, sh, shutdown, w.  All failed commands are symlinks.  (the php command fails because it is not installed in the source filesystem)

updated fs.py + dist fs.pickle = no errors

updated fs.py + createfs generated fs.pickle = previously failed symlink commands succeed. The php command still fails.  No other errors.
